src/Rule/EnumMulti.php

```diff
@@ -53,7 +53,7 @@ public function setParameter($params)
 	public function getMessageParameters()
 	{
 		return array(
-			'values' => implode('|', $this->getParameter()),
+			'value' => implode('|', $this->getParameter()),
```

Comment:
`EnumMulti` seemed to have the value of `{value}` as the value of prameter, but is there a problem here?